src/components/DeeplinkWrapper/deeplinkRoutes.js

```diff
@@ -0,0 +1,47 @@
+export default [
+    {
+        pattern: /\/r\/*/,
```

Comment:
I wonder if it's possible to reduce duplication by re-using the routes listed in [ROUTES.js](https://github.com/Expensify/App/blob/main/src/ROUTES.js)

Reply:
This was fixed